### DISCUSSION TITLE
Font + Typography Resources

### DISCUSSION THREAD

**Discussion by NicholasGannon** [community/community#137500] [Programming Help]
### Body

Hi everyone,

I'm currently working on a repository that provides a curated list of useful **font and typography** resources for designers/developers! I'd really appreciate if you could check it out and give it a star!

https://github.com/NicholasGannon/Font-Resources

Thank you

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by eldarsji** [comment]
Done, I'd be very happy if you could also check my repositories out and star them :D

**Reply by NicholasGannon** [reply]
Thank you so much! Really appreciate it!
